lib/curly/compiler.rb

```diff
@@ -179,7 +179,9 @@ def compile_component(component)
     end
 
     def compile_text(text)
-      output "buffer.safe_concat(#{text.value.inspect})"
+      if non_whitespace_text = text.value.presence
+        output "buffer.safe_concat(#{non_whitespace_text.inspect})"
```

Comment:
This would remove indentional whitespace as well, right?

```html
{{one}}   {{two}}   {{three}}
```

There may be a reason for the whitespace, e.g. when sending plain text emails.